api/applications/applications_controller.go

```diff
@@ -57,6 +58,15 @@ func (ac *applicationController) GetRoutes() models.Routes {
 				Burst: 100,
 			},
 		},
+		models.Route{
+			Path:        rootPath + "/applications/_search",
```

Comment:
is there any reason this could not just be `/search` without the underscore?

Reply:
We have GET, PUT and PATCH methods for /applications/{appName}. If an app has name "search" we could get a potentical conflict if we add a POST method for /applications/{appName}. Currently there is no conflicting methods.

Appnames cannot container underscore, so applications/_search is safe to use.
I copied Elasticsearch API naming "standard". They use underscore when the resource is a verb, like search.